src/Features/Core/Portable/Diagnostics/Analyzers/PreferFrameworkTypeDiagnosticAnalyzerBase.cs

```diff
@@ -56,24 +35,16 @@ public bool OpenFileOnly(Workspace workspace)
             return !(preferTypeKeywordInDeclarationOption == NotificationOption.Warning || preferTypeKeywordInDeclarationOption == NotificationOption.Error ||
                      preferTypeKeywordInMemberAccessOption == NotificationOption.Warning || preferTypeKeywordInMemberAccessOption == NotificationOption.Error);
         }
+        public override DiagnosticAnalyzerCategory GetAnalyzerCategory() 
```

Comment:
Nit: may be missing an empty line above.

Reply:
fixed.